dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/persistence/file/batch/BatchFilePersistenceStrategy.kt

```diff
@@ -6,21 +6,24 @@
 
 package com.datadog.android.core.internal.persistence.file.batch
 
+import com.datadog.android.core.internal.data.upload.DataFlusher
+import com.datadog.android.core.internal.data.upload.Flusher
 import com.datadog.android.core.internal.persistence.DataReader
 import com.datadog.android.core.internal.persistence.DataWriter
 import com.datadog.android.core.internal.persistence.PayloadDecoration
 import com.datadog.android.core.internal.persistence.PersistenceStrategy
 import com.datadog.android.core.internal.persistence.Serializer
 import com.datadog.android.core.internal.persistence.file.FileOrchestrator
+import com.datadog.android.core.internal.persistence.file.advanced.ConsentAwareFileOrchestrator
 import com.datadog.android.core.internal.persistence.file.advanced.ScheduledWriter
 import com.datadog.android.log.Logger
 import java.util.concurrent.ExecutorService
 
 internal open class BatchFilePersistenceStrategy<T : Any>(
-    private val fileOrchestrator: FileOrchestrator,
+    private val fileOrchestrator: ConsentAwareFileOrchestrator,
```

Comment:
We don't want to couple the batching mecanism with the consent tracking one

Reply:
Ok I'll get back to the previous implementation and use that `getFlushableFiles`